File: 1.Layers/1.3Application/1.3.2MainModule/Application.MainModule/DTOs/Mobile/Cortes/UsuariosCorteDTO.cs
/**
 * UsuariosCorteDTO
 * Modelo para el listado de usuarios que 
 * se implementaran en la sección de cortes
 * @Developer: Jorge Omar Tovar Martínez
 * @Date: 11/12/2018
 * @Update: 11/12/2018
 * @company: Neotecks
 */
using Application.MainModule.DTOs.Respuesta;
using System.Collections.Generic;

namespace Application.MainModule.DTOs.Mobile.Cortes
{
    public class UsuariosCorteDTO:RespuestaDto
    {
        /// <summary>
        /// Id de la empresa que corresponde 
        /// </summary>
        public short IdEmpresa { get; set; }
        /// <summary>
        /// Lista de usuarios para cortess
        /// </summary>
        public List<UsuarioDTO> Usuarios { get; set; }
    }
}

File: 1.Layers/1.1Presentation/1.1.2Mobile/app/src/main/java/com/example/neotecknewts/sagasapp/Model/UsuariosCorteDTO.java
/*
 *  UsuariosCorteDTO
 *  Modelo para el listado de usuario para el apartado de anticipos para
 *  especificar quien entrega, este permitira verificar si hay datos y controlar los errores
 *  @developer Jorge Omar Tovar martínez jorge.tovar@neoteck.com.mx
 *  @company Neoteck
 *  @date 12/11/2018
 *  @update 12/11/2018
 */
package com.example.neotecknewts.sagasapp.Model;

import com.example.neotecknewts.sagasapp.Model.Cortes.UsuariosDTO;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class UsuariosCorteDTO extends RespuestaDTO implements Serializable {
    /**
     * Id de la empresa
     */
    @SerializedName("IdEmpresa")
    private int IdEmpresa;
    /**
     * Lista de usuarios
     */
    @SerializedName("Usuarios")
    private List<UsuariosDTO> usuarios;

    /**
     * Constructor de clase
     */
    public UsuariosCorteDTO(){
        this.usuarios = new ArrayList<>();
    }

    public int getIdEmpresa() {
        return IdEmpresa;
    }

    public void setIdEmpresa(int idEmpresa) {
        IdEmpresa = idEmpresa;
    }

    public List<UsuariosDTO> getUsuarios() {
        return usuarios;
    }

    public void setUsuarios(List<UsuariosDTO> usuarios) {
        this.usuarios = usuarios;
    }
}
